packages/camera/ios/Classes/CameraPlugin.m

```diff
@@ -288,9 +288,7 @@ - (void)startVideoRecordingAtPath:(NSString *)path result:(FlutterResult)result
       _eventSink(@{@"event" : @"error", @"errorDescription" : @"Setup Writer Failed"});
       return;
     }
-    [_captureSession stopRunning];
```

Comment:
This stopRunning() / startRunning() pair interfere with the CameraPreview, causing it to freeze momentarily. Are they needed?

Reply:
I think you are right that they are not needed.
I no longer have an iOS setup to test it. Does recording work without them?

Reply:
I checked out their branch and it everything still works when it's removed on my iphone 6.

Reply:
Thanks Maurice for checking that. It also seems to work on my iPhone 7.